src/style.css

```diff
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+.wrapper {
+  margin-top: 20px;
+}
+
+.searchBarWrapper {
```

Comment:
general convention in naming classes in css to use dash-case. In BEM convention will be additional rules.
But not camelCase.
for example search-bar
Just in react apps general class in component can be with CamelCase as component name.